### ISSUE TITLE
Add `on_schema_change` config for incremental models

### ISSUE THREAD

**Issue by awoehrl** [snowplow/dbt-snowplow-web#165]
## Is your feature request related to a problem? Please describe.
The snowplow incremental materialization is great as long as you don't have to change the schema. In case of a schema change it becomes really expensive and/or labour intensive to update tables.

## Describe the solution you'd like
The dbt incremental materialization offers a config to decide on what to do, when the schema of a model changes. This could be a great addition for the snowplow_incremental materialization.

https://docs.getdbt.com/docs/build/incremental-models#what-if-the-columns-of-my-incremental-model-change

**Comment by rlh1994** [comment]
Hi @awoehrl we're actually looking at migrating to using a slightly altered version of the standard `incremental` materialization within a major release or two, so this would be supported by default. We'll let you know when we have a release candidate ready if you want to help us test it out!

**Comment by awoehrl** [comment]
Sounds great, thank you for the info!

**Comment by rlh1994** [comment]
Hey @awoehrl our new pre-release verion https://github.com/snowplow/dbt-snowplow-web/releases/tag/0.14.0-rc1 should allow you to use the standard `incremental` materialization which should contain the `on_schema_change` config. Note that currently it only applies to the core models (not consent) and you need to add the following to your project yaml:

```yml
dispatch:
  - macro_namespace: dbt
    search_order: ['snowplow_utils', 'dbt']
```

It would be great if you could try it out and let us know if it works or any issues you encounter!

**Comment by rlh1994** [comment]
Closed in 0.14.0
